nextflow/assets/multiqc_config.yaml

```diff
@@ -0,0 +1,11 @@
+report_comment: >
+    This report has been generated by the <a href="https://github.com/KwanLab/Autometa" target="_blank">Autometa</a>
+    analysis pipeline. For information about how to interpret these results, please see the
+    <a href="https://github.com/KwanLab/Autometa" target="_blank">documentation</a>.
+report_section_order:
+    software_versions:
+        order: -1000
+    nf-core-autometa-summary:
+        order: -1001
```

Comment:
@WiscEvan not sure what you mean by comment the link.

Reply:
I'm guessing there is a link somewhere in the nf-core docs as to what `report_section_order` is specifying? It would be worth either placing a comment or linking to the nf-core documentation for this config file just as a note for anyone revisiting this code.

Reply:
Started discussion here:
https://github.com/KwanLab/Autometa/issues/163

Reply:
This template has been deleted until/if multiqc is added